.github/dependabot.yml

```
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
```

Would you limit to DS at first?
```suggestion
    directory: "/packages/design-system"
```

---

we use a repository to be aligned on all dependencies why should I limit analysis to design-system ?